examples/transloadit/main.js

```diff
@@ -45,81 +68,149 @@ formUppy.on('upload-error', (file, err) => {
+    hideUploadButton: true,
+  })
+  .use(RemoteSources, { companionUrl: COMPANION_URL })
+  .use(Form, {
+    target: '#dashboard-form',
+    fields: ['message'],
+    triggerUploadOnSubmit: true,
+    submitOnSuccess: true,
+    addResultToForm: true,
+  })
+  .use(Transloadit, {
+    waitForEncoding: true,
+    params: {
+      auth: { key: TRANSLOADIT_KEY },
+      template_id: TEMPLATE_ID,
+    },
+  })
 
 window.formUppyWithDashboard = formUppyWithDashboard
 
-const dashboard = robodog.dashboard('#dashboard', {
+/**
+ * Dashboard
+ */
+
+const dashboard = new Uppy({
   debug: true,
-  waitForEncoding: true,
-  note: 'Images will be resized with Transloadit',
-  params: {
-    auth: { key: TRANSLOADIT_KEY },
-    template_id: TEMPLATE_ID,
+  autoProceed: false,
+  restrictions: {
+    allowedFileTypes: ['.png'],
   },
 })
+  .use(Dashboard, {
+    inline: true,
+    target: '#dashboard',
+    note: 'Only PNG files please!',
+  })
+  .use(RemoteSources, { companionUrl: COMPANION_URL })
+  .use(Webcam, { target: Dashboard })
+  .use(ImageEditor, { target: Dashboard })
+  .use(Transloadit, {
+    waitForEncoding: true,
+    params: {
+      auth: { key: TRANSLOADIT_KEY },
+      template_id: TEMPLATE_ID,
+    },
+  })
 
 window.dashboard = dashboard
 
-/**
- * robodog.modal
- */
+// /**
+//  * Dashboard Modal
+//  */
 
-function openModal () {
-  robodog.pick({
-    restrictions: {
-      allowedFileTypes: ['.png'],
-    },
+const dashboardModal = new Uppy({
+  debug: true,
+  autoProceed: false,
+})
+  .use(Dashboard, { closeAfterFinish: true })
+  .use(RemoteSources, { companionUrl: COMPANION_URL })
+  .use(Webcam, { target: Dashboard })
+  .use(ImageEditor, { target: Dashboard })
+  .use(Transloadit, {
     waitForEncoding: true,
     params: {
       auth: { key: TRANSLOADIT_KEY },
       template_id: TEMPLATE_ID,
     },
-    providers: [
-      'webcam',
-    ],
-    // if providers need custom config
-    // webcam: {
-    //   option: 'whatever'
-    // }
-  }).then(console.log, console.error)
+  })
+
+dashboardModal.on('complete', ({ transloadit, successful, failed }) => {
+  console.log(transloadit)
+  console.log(successful)
+  console.error(failed)
+})
+
+function openModal () {
+  dashboardModal.getPlugin('Dashboard').openModal()
 }
 
 window.openModal = openModal
 
-/**
- * robodog.upload
- */
+// /**
+//  * uppy.upload (files come from input[type=file])
+//  */
 
-window.doUpload = (event) => {
-  const resultEl = document.querySelector('#upload-result')
-  const errorEl = document.querySelector('#upload-error')
-  robodog.upload(event.target.files, {
+const uppyWithoutUI = new Uppy({
+  debug: true,
+  restrictions: {
+    allowedFileTypes: ['.png'],
+  },
+})
+  .use(Transloadit, {
     waitForEncoding: true,
     params: {
       auth: { key: TRANSLOADIT_KEY },
       template_id: TEMPLATE_ID,
     },
-  }).then((result) => {
+  })
+  .use(ProgressBar, { target: '#upload-progress' })
+
+window.doUpload = (event) => {
```

Comment:
Probably needs explanation that this requires HTML as well, particularly a `input type="file"`